modules/installation-and-upgrade/pages/install-vm.adoc

```diff
@@ -115,10 +115,10 @@ SUSEConnect -e <EMAIL_ADDRESS> -r <SUSE_MANAGER_CODE> \
 +
 
 ----
-SUSEConnect -p sle-module-basesystem/15.5/x86_64
-SUSEConnect -p sle-module-server-applications/15.5/x86_64
-SUSEConnect -p sle-module-web-scripting/15.5/x86_64
-SUSEConnect -p sle-module-suse-manager-server/<productnumber>/x86_64
```

Comment:
on the proxy string for suseconnect, I don't know why you removed the productnumber and the arch.

looking at parameters.yml the variable could be
smrproductnumber

Reply:
Was introduced with cherry-picking.  Reverted.  I'll let Ornela and Joseph decide whether we want to use smrproductnumber here (iirc, this was intended for SUSE Manager Retail).

Reply:
We have decided that his line stays as it was in all previous versions of the documentation.